api/user/auth.service.ts

```diff
@@ -16,6 +16,27 @@ export interface AuthenticateReturn {
   message?: string
 }
 
+type UserReturned = {
```

Comment:
Normally, these types are automatically generated using `yarn graphql:generate`

Reply:
The `User` type used in auth service is of `'@prisma/client'` type, which has a different type than the `User` defined in graphql schema. 
error
`Type 'User' is missing the following properties from type 'User': documentsRaw, documents, groupsts(2739)`
So we have to overwrite the User type...

Reply:
> The User type used in auth service is of '@prisma/client' type, which has a different type than the User defined in graphql schema.

That's right. However, in https://github.com/JabRef/JabRefOnline/blob/8a922d67a85d65d8c260e87de500c67b0452f31d/graphql.config.js#L17 we tell the graphql code generator that it should expect a prisma user to be returned whenever it sees a User type in the graphql schema. The transformation of a prisma user to a valid graphql type is handled by https://github.com/JabRef/JabRefOnline/blob/8a922d67a85d65d8c260e87de500c67b0452f31d/api/user/resolvers.ts#L103-L136

But probably I'm misunderstanding something...

Reply:
@mohit038 I think I found a way to reuse the auto-generated types. At least it is working for me now. I've pushed the changes to this branch. Please have a look.